docs/index.md

```diff
@@ -72,33 +72,73 @@ connection "digitalocean" {
 
 ### Example Configurations
 
-- Connect to a single account:
-
-  ```hcl
-  connection "digitalocean_my_account" {
-    plugin = "digitalocean"
-    token  = "1646968370949-df954218b5da5b8614c85cc454136b27"
-  }
-  ```
-
-- Create connections to multiple accounts:
-
-  ```hcl
-  connection "account_aaa" {
-    plugin    = "digitalocean"
-    token     = "1646968370949-df954218b5da5b8614c85cc4541abcde"
-  }
-
-  connection "account_bbb" {
-    plugin    = "digitalocean"
-    token     = "1646968370949-df954218b5da5b8614c85cc4541fghij"
-  }
-
-  connection "account_ccc" {
-    plugin    = "digitalocean"
-    token     = "1646968370949-df954218b5da5b8614c85cc4541klmno"
-  }
-  ```
+Connect to a single account:
+
+```hcl
+connection "digitalocean_my_account" {
+  plugin = "digitalocean"
+  token  = "1646968370949-df954218b5da5b8614c85cc454136b27"
+}
+```
+
+## Multi-Account Connections
+
+You may create multiple digitalocean connections:
+
+```hcl
+connection "do_dev" {
+  plugin    = "digitalocean"
+  token     = "1646968370949-df954218b5da5b8614c85cc4541abcde"
+}
+
+connection "do_qa" {
+  plugin    = "digitalocean"
+  token     = "1646968370949-df954218b5da5b8614c85cc4541fghij"
+}
+
+connection "do_prod" {
+  plugin    = "digitalocean"
+  token     = "1646968370949-df954218b5da5b8614c85cc4541klmno"
+}
+```
+
+Each connection is implemented as a distinct [Postgres schema](https://www.postgresql.org/docs/current/ddl-schemas.html). As such, you can use qualified table names to query a specific connection:
+
+```sql
+select * from do_qa.digitalocean_project
+```
+
+You can multi-account connections by using an [**aggregator** connection](https://steampipe.io/docs/using-steampipe/managing-connections#using-aggregators). Aggregators allow you to query data from multiple connections for a plugin as if they are a single connection.
+
+```hcl
+connection "do_all" {
+  plugin      = "digitalocean"
+  type        = "aggregator"
+  connections = ["do_dev", "do_qa", "do_prod"]
+}
+```
+
+Querying tables from this connection will return results from the `do_dev`, `do_qa`, and `do_prod` connections:
+
+```sql
+select * from do_all.digitalocean_project
+```
+
+Alternatively, can use an unqualified name and it will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphbetically, so that it is the first connection in the search path (i.e. `do_all` comes before `do_dev`):
```

Comment:
```suggestion
Alternatively, you can use an unqualified name and it will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphabetically, so that it is the first connection in the search path (i.e. `do_all` comes before `do_dev`):
```